extensions-contrib/statsd-emitter/src/main/java/org/apache/druid/emitter/statsd/StatsDEmitter.java

```diff
@@ -174,6 +177,59 @@ public void emit(Event event)
       } else {
         log.debug("Service=[%s], Metric=[%s] has no StatsD type mapping", service, metric);
       }
+    } else if (event instanceof AlertEvent && config.isDogstatsd()) {
```

Comment:
Do you think it might be reasonable to allow users to enable metrics and events separately? eg, to turn off event emitting if you're not using it?

Reply:
Do you mean enable event emitting when you are not using DogStatsD?

Reply:
sounds reasonable, should we leave it off by default then?

Reply:
I added an option to turn this on, and left alert events off by default. Updated docs as well.